newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -30,14 +32,39 @@ export default function SummaryPage() {
     setSubmitting(false);
   };
 
+  const sendSummaryEmails = async () => {
```

Comment:
I think you can use pedro's new `client.useBackend` hook here

Reply:
It's not Pedro's, it's newdle's 😉